vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -274,7 +274,17 @@ public static function should_run_connection_pilot(): bool {
 	 * @return bool True if a reconnect should be attempted
 	 */
 	public static function should_attempt_reconnection( \WP_Error $error = null ): bool {
-		// The constant is deprecated, but keeping this check for historical reasons
+		// TODO: Only attempting to reconnect on new sites. We can remove this code after ramp-up
+		if ( ! is_multisite() && defined( 'VIP_GO_APP_ID' ) && VIP_GO_APP_ID < 3750 ) {
```

Comment:
I think this needs to be the other way around? 
```suggestion
		if ( ! is_multisite() && defined( 'VIP_GO_APP_ID' ) &&  3750 < VIP_GO_APP_ID  ) {
```

Reply:
I think it is correct as is. The function returns `false` in case we don't want to attempt reconnection. In this case, we only want to attempt it for "new" sites, that is, sites that have an ID bigger than 3750.

Reply:
ahh, I misread the next line :) Please disregard.